admin/client/components/ImagesDiff.js

```diff
@@ -10,127 +10,127 @@ import Rcslider from 'rc-slider';
 const bgImage = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUAQMAAAC3R49OAAAABlBMVEX5+fn///8pDrwNAAAAFElEQVQI12NgsP/AQAz+f4CBGAwAJIIdTTn0+w0AAAAASUVORK5CYII=';
```

Comment:
Is there any reason why we should fix the indentation in two stages?

Reply:
Just to be clear, the second stage causes by updating .eslintrc with `{ parserOptions: "ecmaVersion": 9 } `.  

I could merge them into single commit then.

Reply:
They are squashed into https://github.com/twreporter/keystone/pull/194/commits/4de499a23b7f3b81d8f58641f7ab3ce1b0591b10.